source/include/sigv4.h

```diff
@@ -150,7 +150,16 @@ typedef enum SigV4Status
      * Functions that may return this value:
      * - #SigV4_AwsIotDateToIso8601
      */
-    SigV4ISOFormattingError
+    SigV4ISOFormattingError,
+
+    /**
+     * @brief The maximum number of header parameters was exceeded while parsing
+     * the http header string input parameter.
+     *
```

Comment:
If there is an upper limit on the number of header parameters supported for parsing by the library specified in a macro, it can be mentioned here.